sklearn/covariance/outlier_detection.py

```diff
@@ -28,26 +29,32 @@ class OutlierDetectionMixin(object):
         The amount of contamination of the data set, i.e. the proportion
         of outliers in the data set.
 
+    raw_values : bool
```

Comment:
can we rename this `raw_decision` or something? The name was relevant in a `decision_function` context and here is underspecified